cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -539,33 +574,15 @@
         return Image(**data)
 
 
-class CustomActionSchema(BaseSchema):
```

Comment:
Are we removing custom action (pre/post install scriopt) support? If memory serves, we are just removing the `run_as` field